packages/x-adapter-platform/src/schemas/models/facet.schema.ts

```diff
@@ -2,17 +2,17 @@ import { createMutableSchema, Schema } from '@empathyco/x-adapter';
 import {
   EditableNumberRangeFacet,
   HierarchicalFacet,
-  HierarchicalFilter,
   NumberRangeFacet,
   SimpleFacet
 } from '@empathyco/x-types';
-import { numberFilterMutableSchema } from '../filters/number-filter.schema';
-import { simpleFilterMutableSchema } from '../filters/simple-filter.schema';
-import { PlatformFacet, PlatformHierarchicalFilter } from '../../types/models/facet.model';
+import { PlatformFacet } from '../../types/models/facet.model';
 import { getFacetConfig } from '../facets/utils';
 import { FacetsConfig } from '../facets/types';
+import { hierarchicalFilterSchema } from './filters/hierarchical-filter.schema';
+import { numberFilterSchema } from './filters/number-filter.schema';
+import { simpleFilterSchema } from './filters/simple-filter.schema';
```

Comment:
Could these 3 imports be bundled together?

```suggestion
import { hierarchicalFilterSchema, numberFilterSchema, simpleFilterSchema } from './filters';
```

Reply:
but usually we don't want to import from barrels, do we?